components/Header/Header.tsx

```diff
@@ -89,17 +89,17 @@ export const Header = ({ jobsAvailable = 0 }: HeaderProps) => {
                     />
                 </div>
             </div>
+            <MobileMenuButton
```

Comment:
Moving the mobile menu button ensures that the tabbing order is correct when navigating the mobile menu with the keyboard (the mobile menu shows at some desktop screen sizes so this is possible / likely)